DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java

```diff
@@ -139,12 +140,26 @@ public void postInit(Scene scene) {
             }
         });
 
+        String deviceFactorSuffix = Services.get(DeviceService.class)
+                .map(s -> {
```

Comment:
Much cleaner solution for search is a combination of filter and findFirst

Reply:
But existing code should work

Reply:
Would you say this is faster?
It requires a global String though. I don't see any advantage on it.

```
String deviceFactorSuffix = "";
        if (Platform.isAndroid()) {
            Services.get(DeviceService.class)
                .filter(s -> s.getModel() != null)
                .ifPresent(s -> {
                    for (String device : DevoxxSettings.DEVICES_WITH_SANS_CSS) {
                        if (s.getModel().toLowerCase(Locale.ROOT).startsWith(device)) {
                            deviceFactorSuffix = "_sans";
                        }
                    }
                });
        }
```

Reply:
No.. I meant the device search.  
Something like.. ```DevoxxSettings.DEVICES_WITH_SANS_CSS.stream.filter.findFirst```, than you can do ```flatMap``` on the service itself. After that you get an ```Optional```, which can be converted into ```"_sans"``` or ```""```

All that will look cleaner IMO, but it should work fine as it is now

Reply:
Once we have streams running, we'll apply that :)

Reply:
Ah... not yet.. not yet 👍